operator/controllers/keptntask/controller.go

```diff
@@ -83,3 +90,26 @@ func (r *KeptnTaskReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&batchv1.Job{}).
 		Complete(r)
 }
+
+func (r *KeptnTaskReconciler) IsJobRunning(ctx context.Context, task klcv1alpha1.KeptnTask, namespace string) (bool, error) {
+	jobList := &batchv1.JobList{}
+
+	jobLabels := client.MatchingLabels{}
+	for k, v := range createKeptnLabels(task) {
+		jobLabels[k] = v
+	}
+
+	if len(jobLabels) == 0 {
+		return false, fmt.Errorf("no labels found for task: %s", task.Name)
+	}
+
+	if err := r.Client.List(ctx, jobList, client.InNamespace(namespace), jobLabels); err != nil {
+		return false, err
+	}
+
+	if len(jobList.Items) > 0 {
```

Comment:
Is it enough to check if there is at least one? Should we check also if it is not terminating? 🤔

Reply:
I think so, because this would mean that we already created it ...